cristin-biobank/src/main/java/no/unit/nva/biobank/model/cristin/CristinAssocProjectForBiobank.java

```diff
@@ -0,0 +1,48 @@
+package no.unit.nva.biobank.model.cristin;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.net.URI;
+import java.util.Map;
+
+public class CristinAssocProjectForBiobank {
+
+    public static final String CRISTIN_PROJECT_IDENTIFICATOR = "cristin_project_id";
+    public static final String CRISTIN_PROJECT_ID = "url";
+    public static final String CRISTIN_PROJECT_TITLE = "title";
+
+    @JsonProperty(CRISTIN_PROJECT_ID)
```

Comment:
This points to string value "url" and is hard to understand. Use better names describing the constants.

```
public static final String CRISTIN_PROJECT_ID = "cristin_project_id";
public static final String CRISTIN_PROJECT_URL = "url";
public static final String CRISTIN_PROJECT_TITLE = "title";
```

Or something similar. See earlier comment